test/features/email/presentation/controller/single_email_controller_test.dart

```diff
@@ -252,4 +261,39 @@ void main() {
       },
     );
   });
+
+  group('calendar event reply -', () {
+    final blobId = Id('abc123');
+    final calendarEvent = CalendarEvent();
+
+    setUp(() {
+      when(mailboxDashboardController.selectedEmail).thenReturn(Rxn(null));
+      when(mailboxDashboardController.emailUIAction).thenReturn(Rxn(null));
+      when(mailboxDashboardController.viewState).thenReturn(Rx(Right(UIState.idle)));
+      singleEmailController.onInit();
+    });
+
+    group('accept:', () {
```

Comment:
idem

Reply:
Done in:
- https://github.com/linagora/tmail-flutter/pull/2803